packages/ag-charts-community/src/scale/continuousScale.ts

```diff
@@ -55,42 +54,42 @@ export abstract class ContinuousScale<D extends number | Date, I = number> exten
         return rangeDistance / Math.max(1, bands);
     }
 
-    convert(value: D, options?: { clamp?: boolean }) {
+    convert(value: D | number, options?: { clamp?: boolean }) {
         const { domain } = this;
         if (!domain || domain.length < 2) {
             return NaN;
         }
 
+        const { transform } = this;
         const clamp = options?.clamp ?? this.defaultClamp;
-        const d0 = Number(this.transform(domain[0]));
-        const d1 = Number(this.transform(domain[1]));
-        const x = Number(this.transform(value));
-
-        const { range } = this;
-        const [r0, r1] = range;
+        const d0 = (transform ? transform(domain[0])?.valueOf() : domain[0]?.valueOf()) ?? NaN;
```

Comment:
You could do `this.transform?.(...)` rather than binding

Reply:
Reworked this aspect, thanks for the feedback.